<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-5F10D1SQ" ignored="0" name="Default test" tests="5" failures="1" timestamp="2021-08-25T01:49:14 IST" time="1.452" errors="0">
  <testcase name="@AfterMethod getResult" time="0.014" classname="com.thritybees.extentreports.ExtentReportV4">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.thritybees.extentreports.ExtentReportV4.getResult(ExtentReportV4.java:84)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:701)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:677)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:661)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod tearDown" time="-1.629836354372E9" classname="com.thirtybees.test.CreateAccountTest">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@AfterMethod getResult" time="-1.629836354375E9" classname="com.thritybees.extentreports.ExtentReportV4">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod tearDown" time="-1.629836354385E9" classname="com.thirtybees.test.CreateAccountTest">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@AfterMethod getResult" time="-1.629836354387E9" classname="com.thritybees.extentreports.ExtentReportV4">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod tearDown" time="-1.62983635444E9" classname="com.thirtybees.test.CreateAccountTest">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@AfterMethod getResult" time="-1.629836354445E9" classname="com.thritybees.extentreports.ExtentReportV4">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod tearDown" time="-1.629836354469E9" classname="com.thirtybees.test.CreateAccountTest">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@AfterMethod getResult" time="-1.629836354472E9" classname="com.thritybees.extentreports.ExtentReportV4">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterTest endReport" time="-1.629836354487E9" classname="com.thritybees.extentreports.ExtentReportV4">
    <skipped/>
  </testcase> <!-- @AfterTest endReport -->
  <testcase name="validateBlankLoginCredential" time="0.684" classname="com.thirtybees.test.CreateAccountTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.thritybees.pages.CreateAccountPage.verifyBlankLoginCredential(CreateAccountPage.java:83)
at com.thirtybees.test.CreateAccountTest.validateBlankLoginCredential(CreateAccountTest.java:72)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateBlankLoginCredential -->
  <testcase name="validateInvalidateLoginCredential" time="0.0" classname="com.thirtybees.test.CreateAccountTest">
    <skipped/>
  </testcase> <!-- validateInvalidateLoginCredential -->
  <testcase name="validateValidLoginCredential" time="0.0" classname="com.thirtybees.test.CreateAccountTest">
    <skipped/>
  </testcase> <!-- validateValidLoginCredential -->
  <testcase name="verifyInValidRegistration" time="0.0" classname="com.thirtybees.test.CreateAccountTest">
    <skipped/>
  </testcase> <!-- verifyInValidRegistration -->
  <testcase name="verifyRegistration" time="0.0" classname="com.thirtybees.test.CreateAccountTest">
    <skipped/>
  </testcase> <!-- verifyRegistration -->
</testsuite> <!-- Default test -->
